get-deps

```diff
@@ -96,6 +96,11 @@ fi
 
 if test -e /etc/arch-release ; then
   PACMAN="$SUDO pacman"
+  
```

Comment:
I'm not an Arch expert, but https://archlinux.org/groups/x86_64/base-devel/ suggests that `base-devel` is part of arch itself, so I think you could move this to the list below

Reply:
```suggestion
```

Reply:
@wez Yeah, I just assumed the existing list worked on arch so I didn't want to install anything extra on vanilla arch if it was only necessary on manjaro